02.calendar/calendar.rb

```diff
@@ -3,27 +3,27 @@
 require 'date'
 require 'optparse'
 
-todays = Date.today
-this_year = todays.year
-this_month = todays.month
+today = Date.today
+year = today.year
+month = today.month
 
 opt = OptionParser.new
-opt.on('-m [manth]', '月を指定します-m') { |m| this_month = m.to_i }
-opt.on('-y [year]', '年を指定します -y') { |y| this_year = y.to_i }
+opt.on('-m [month]', '月を指定します -m') { |m| month = m.to_i }
+opt.on('-y [year]', '年を指定します -y') { |y| year = y.to_i }
 opt.parse!(ARGV)
 
-first_day = Date.new(this_year, this_month, 1)
-last_day = Date.new(this_year, this_month, -1)
+first_date = Date.new(year, month, 1)
+last_date = Date.new(year, month, -1)
 
-puts "      #{this_month}月 #{this_year}"
+puts "      #{month}月 #{year}"
 puts '日 月 火 水 木 金 土'
 
-first_day.wday.times do
+first_date.wday.times do
   print '   '
 end
 
-(first_day..last_day).each do |date|
-  print(sprintf("%2d", date.day.to_s) + ' ')
+(first_date..last_date).each do |date|
+  print "#{date.day.to_s.rjust(2)} "
   if date.saturday?
     print("\n")
```

Comment:
```suggestion
    print "\n"
```

細かいですが、putsやprintで`()`を使うのか使わないかを統一した方が一貫性があって良いと思います。